src/main/java/org/folio/linked/data/controller/AuthorityAssignmentController.java

```diff
@@ -4,15 +4,15 @@
 
 import lombok.RequiredArgsConstructor;
 import org.folio.linked.data.domain.dto.AssignmentCheckDto;
-import org.folio.linked.data.rest.resource.LinkedDataApi;
+import org.folio.linked.data.rest.resource.AuthorityApi;
 import org.folio.linked.data.service.resource.AssignAuthorityTarget;
 import org.folio.linked.data.service.resource.ResourceMarcAuthorityService;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequiredArgsConstructor
-public class AuthorityAssignmentController implements LinkedDataApi {
+public class AuthorityAssignmentController implements AuthorityApi {
```

Comment:
@pkjacob @PBobylev @AndreiBordak I've added this change after rebase
please let me know if you are ok with it